src/i_sound.c

```diff
@@ -493,6 +493,7 @@ void I_BindSoundVariables(void)
     extern char *snd_dmxoption;
     extern int use_libsamplerate;
     extern float libsamplerate_scale;
+    extern int fluidsynth_force_off; // force Fluidsynth off
```

Comment:
Please don't add `extern` definitions to .c files. The compiler can't check them and it's easy to get undefined behavior. You should also move the ones that are already there.